server/channels/api4/access_control.go

```diff
@@ -777,3 +779,49 @@ func convertToVisualAST(c *Context, w http.ResponseWriter, r *http.Request) {
 		c.Logger.Warn("Error while writing response", mlog.Err(err))
 	}
 }
+
+func createAccessControlSyncJob(c *Context, w http.ResponseWriter, r *http.Request) {
+	var jobData map[string]string
```

Comment:
jobData is a user provided value. If I understood the code right, we are only expecting the policy ID. But the user can add any other value there.

Looking at the code, this will go unvalidated (apart of the policy id) all the way to store the job in the DB.

If that is not intended, I would extract the policy id (be it here or in the app layer, 0/5), and create a new jobData object with only the information we need.